### DISCUSSION TITLE
plots locked

### DISCUSSION THREAD

**Discussion by olivierLA75** [Chia-Network/bladebit#76] [Q&A]
I'm not sure why but the plots I'm making with Bladebit are locked and I can't even move them? any idea on how to fix this?
thanks
Olivier
<img width="476" alt="Screen Shot 2021-08-22 at 11 25 02 PM" src="https://user-images.githubusercontent.com/83734065/130400494-87c27636-5b04-4943-82f8-75e01ff2d9a1.png">

**Comment by harold-b** [comment]
If you are still experiencing this issue: Can you open a terminal in that folder and run `ls -la`?
